samples/zigbee/shell/prj.conf

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_NCS_SAMPLES_DEFAULTS=y
+
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_SERIAL=y
+CONFIG_GPIO=y
+
+# Make sure printk is not printing to the UART console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=2048
+CONFIG_MAIN_THREAD_PRIORITY=7
+
+CONFIG_ZIGBEE=y
+CONFIG_ZIGBEE_APP_UTILS=y
+CONFIG_ZIGBEE_ROLE_ROUTER=y
+
+# Enable DK LED and Buttons library
+CONFIG_DK_LIBRARY=y
+
+# This example requires more workqueue stack
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+# Enable nRF ECB driver
+CONFIG_CRYPTO=y
+CONFIG_CRYPTO_NRF_ECB=y
+CONFIG_CRYPTO_INIT_PRIORITY=80
+
+# Networking
+CONFIG_NET_IPV6_MLD=n
+CONFIG_NET_IPV6_NBR_CACHE=n
+CONFIG_NET_IPV6_RA_RDNSS=n
+CONFIG_NET_IP_ADDR_CHECK=n
+CONFIG_NET_UDP=n
+
+# Zigbee shell
+CONFIG_ZIGBEE_SHELL=y
+CONFIG_ZIGBEE_SHELL_DEBUG_CMD=y
+CONFIG_ZIGBEE_SHELL_ENDPOINT=10
```

Comment:
@sebastiandraus : Shall we use endpoint 64 or it does not matter in NCS?

Reply:
Doesn't matter but I'd use endpoint 64 to keep this similar to sample from nRF5 SDK

Reply:
Changed to 64.